### PR TITLE
feat: new poster html

### PR DESCRIPTION
**Pull request by sexta13**
### What does it do?
 - Updates poster with new layout. -  https://trello.com/c/ODDwDtIa/35-atualizar-cartaz-html-com-novo-layout

### CODE REVIEW

**Comment by sexta13** [comment]
image

This is how it looks when sending to print

File: src/pages/QueuePoster/QueuePoster.js [hunk 1]
```diff
@@ -40,40 +42,32 @@ function QueuePoster({
   }, [queueId]);
 
   if (loading) return <Loader />;
-
+  console.log(queue);
```
Comment (xicosantos98) [inline src/pages/QueuePoster/QueuePoster.js:45]: Remove this console.log

File: src/i18n.js [hunk 2]
```diff
@@ -306,7 +306,11 @@ i18n
           "admin#queuePoster_queueCode": "Código da Fila",
           "admin#queuePoster_enterQueueWith":
             "Entre &nbsp <b>nafila</b> &nbsp com...",
-          "admin#queuePoster_brandSlogan": "<b>naFila</b>, sem filas."
+          "admin#queuePoster_brandSlogan": "<b>naFila</b>, sem filas.",
+          "admin#queuePoster_mark_your_time_in_queue":
+            "MARQUE A SUA VEZ NA FILA",
+          "admin#send_free_sms_queue":
+            "Envie <b>SMS GRÁTIS</b> com <b><u>nafila {{queueCode}}</u></b> para o <b>4902</b>"
```
Comment (carlosouro) [inline src/i18n.js:313]: Falei com a Rita do design.
Retira o underline e deixa so bold no texto da SMS.

File: src/pages/QueuePoster/QueuePoster.js (hunk 1 above)
Comment (sexta13) [reply src/pages/QueuePoster/QueuePoster.js:45]: ups :P

File: src/i18n.js (hunk 2 above)
Comment (sexta13) [reply src/i18n.js:313]: oki

File: src/pages/QueuePoster/QueuePoster.js (hunk 1 above)
Comment (sexta13) [reply src/pages/QueuePoster/QueuePoster.js]: done

File: src/i18n.js (hunk 2 above)
Comment (sexta13) [reply src/i18n.js]: done

**Comment by sexta13** [comment]
after last changes, looks like this
image

**Pull request merged**